src/components/Alert/Alert.tsx

```
@@ -0,0 +1,75 @@
+import React from "react";
+import { CheckCircle, Info, WarningCircle, XCircle } from "phosphor-react";
+
+type AlertProps = {
+    title?: string;
```

ESLint

[error] Delete `··`

(prettier/prettier)